dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -984,6 +1025,13 @@
           "items": {
             "$ref": "#/definitions/HealthcareEntityLink"
           }
+        },
+        "EntityComponentInformation": {
```

Comment:
Where is this array being used? Is the intention to add it to `HealthcareEntitiesDocumentResultWithDocumentDetectedLanguage`?